### PR TITLE
feat(handlers): Wrapping json rpc handlers to catch panics

### PR DESCRIPTION
**Pull request by arhamj**
Summary: Added a wrapped handler which logs and returns 500 in case of errors

### CODE REVIEW

**Review by abdulazeem-tk4vr — APPROVED** [review]

File: src/api.ts [hunk 1]
```diff
@@ -709,6 +711,10 @@ async function injectAndRecordTx(
         }
       })
       .catch((e: Error) => {
+        if (e.message.includes('timeout')) {
+          // TODO: add to blacklist with a TTL and use this blacklist to avoid bad node selction for inject.
```
Comment (mssabr01) [inline src/api.ts:715]: Remove TODO and create a task for this

**Review by mssabr01 — COMMENTED** [review]
Minor comment, please make sure the TODO is removed and a task is created out of it.

**Review by mhanson-github — COMMENTED** [review]
test

File: src/api.ts [hunk 2]
```diff
@@ -647,7 +647,9 @@ async function injectAndRecordTx(
     const aalgTime = injectStartTime - startTime
     console.log(`injecting tx to`, `${baseUrl}/${injectEndpoint}`, injectStartTime)
     axios
```
Comment (mhanson-github) [inline src/api.ts:649]: Another test

**Pull request merged**